src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApiTest.java

```diff
@@ -0,0 +1,172 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import feign.FeignException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.config.ServiceConfigProvider;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.doThrow;
+
+@ExtendWith(MockitoExtension.class)
+class CcdApiTest {
+    @Mock
+    private CoreCaseDataApi feignCcdApi;
+
+    @Mock
+    private CcdAuthenticatorFactory authenticatorFactory;
+
+    @Mock
+    private ServiceConfigProvider serviceConfigProvider;
+
+    @Mock
+    private CcdAuthenticator ccdAuthenticator;
+
+    @Mock
+    private UserDetails userDetails;
+
+    @Mock
+    private CaseDetails caseDetails;
+
+    @Mock
+    private StartEventResponse startEventResponse;
+
+    private CcdApi ccdApi;
+
+    @BeforeEach
+    public void setUp() {
+        ccdApi = new CcdApi(feignCcdApi, authenticatorFactory, serviceConfigProvider);
+    }
+
+    @Test
+    void should_rethrow_feign_exception_in_startAttachScannedDocs() {
+        // given
+        given(caseDetails.getJurisdiction()).willReturn("BULKSCAN");
+        given(caseDetails.getCaseTypeId()).willReturn("caseTypeId");
+        given(authenticatorFactory.createForJurisdiction(anyString())).willReturn(ccdAuthenticator);
+        given(ccdAuthenticator.getServiceToken()).willReturn("serviceToken");
+        doThrow(
+            new FeignException.BadRequest("Msg", "Body".getBytes()))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Msg" appears 8 times in this file; the first occurrence is on line 59](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4567585)